client/js/components/user/CustomerSettings/CustomerSettingsContact.vue

```diff
@@ -0,0 +1,226 @@
+          v-text="contact.attributes.title" />
+        <span
+          class="block"
+          v-text="contact.attributes.phoneNumber" />
+        <span
+          class="block"
+          v-text="contact.attributes.eMailAddress" />
+        <span
+          class="block"
+          v-html="contact.attributes.text" />
+        <span v-text="Translator.trans('customer.contact.visibleText', {isVisible: contact.attributes.visible})" />
+      </template>
+      <template v-slot:form>
+        <div
+          id="contactForm"
+          data-dp-validate="contactData">
+          <dp-input
+            id="contactTitle"
+            v-model="customerContact.title"
+            class="u-mb-0_75"
+            data-cy="contactTitle"
+            data-dp-validate-error="error.title"
+            :placeholder="Translator.trans('customer.contact.title')"
+            required
+            type="text" />
+          <dp-input
+            id="phoneNumber"
+            v-model="customerContact.phoneNumber"
+            autocomplete="tel"
+            class="u-mb-0_75"
+            data-cy="phoneNumber"
+            data-dp-validate-error="error.phone_or_email"
+            pattern="^(\+?)(-| |[0-9]|\(|\))*$"
+            :placeholder="Translator.trans('customer.contact.phone_number')"
+            :required="customerContact.eMailAddress === ''"
+            type="tel" />
+          <dp-input
+            id="emailAddress"
+            v-model="customerContact.eMailAddress"
+            autocomplete="email"
+            class="u-mb-0_75"
+            data-cy="emailAddress"
+            data-dp-validate-error="error.phone_or_email"
+            :placeholder="Translator.trans('email.address')"
+            :required="customerContact.phoneNumber === ''"
+            type="email" />
+          <dp-editor
+            id="supportText"
+            class="u-mb-0_75"
+            v-model="customerContact.text"
+            hidden-input="supportText"
+            :toolbar-items="{
+              fullscreenButton: true,
+              headings: [2,3,4],
+              linkButton: true
+            }" />
+          <dp-checkbox
+            id="contactVisible"
+            v-model="customerContact.visible"
+            data-cy="contactVisible"
+            :label="{
+              text: Translator.trans('customer.contact.visible')
+            }" />
+        </div>
+      </template>
+    </dp-editable-list>
+  </div>
+</template>
+
+<script>
+import { DpCheckbox, DpEditableList, DpEditor, DpInput, dpValidateMixin } from '@demos-europe/demosplan-ui'
+import { mapActions, mapMutations, mapState } from 'vuex'
+
+export default {
+  name: 'CustomerSettingsContact',
+
+  components: {
+    DpCheckbox,
+    DpEditableList,
+    DpEditor,
+    DpInput
+  },
+
+  mixins: [dpValidateMixin],
+
+  data () {
+    return {
+      customerContact: {
+        title: '',
+        phoneNumber: '',
+        eMailAddress: '',
+        text: '',
+        visible: false
+      },
```

Comment:
You could create a global const customerContactDefault with this data and assign it here.
And then in `resetForm()` use it again. like `this.customerContact = customerContactDefault`